src/ucp/core/ucp_request.h

```diff
@@ -83,6 +92,14 @@ enum {
     UCP_RDESC_ALL_LIST  = 1
 };
 
+/**
+ * Send request type
+ */
+typedef enum {
+    UCP_REQUEST_TYPE_SEND_AM = 0,
```

Comment:
pls align by =, can we define this as flags and pack to request->flags instead of adding new field?